go/ct/driver/run.go

```diff
@@ -216,6 +224,13 @@ func runTest(input *st.State, evm ct.Evm, filter *regexp.Regexp) error {
 
 	result, err := evm.StepN(input.Clone(), 1)
 	defer result.Release()
+
+	targetError := &vm.ErrUnsupportedRevision{}
+	if errors.As(err, &targetError) {
+		numUnsupportedTests.Add(1)
+		return nil
+	}
```

Comment:
nit: maybe, instead of adding an out-parameter (`numUnsupportedTests`) to this function, all errors should just be returned the caller of this function should check for the the unsupported revision error.